russian/sleep/zefir.md

```diff
@@ -1,5 +1,6 @@
 Вы просыпаетесь и понимаете что всё ещё находитесь в комнате, стены которой сделаны из зефира. 
-Стены очень приятно пахнут. Интересно, какие они на вкус?
+-Стены очень приятно пахнут. Интересно, какие они на вкус?
```

Comment:
It looks like you've added a minus sign or hyphen at the beginning of this line. Was that a mistake? (Neither minus sign in the red line nor the plus sign in the green line are actually part of the file - they indicate that the line was removed and added respectively. So this change shows that a line without a hyphen was removed and a similar line with a hyphen was added.)